src/settings-ui/Settings.UI/Views/ShellPage.xaml

```diff
@@ -145,6 +146,10 @@
                         x:Uid="Feedback_NavViewItem"
                         Icon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xED15;}"
                         Tapped="FeedbackItem_Tapped" />
+                    <NavigationViewItem
+                        x:Uid="ExitPT_NavViewItem"
+                        Icon="{ui:FontIcon FontFamily={StaticResource SymbolThemeFontFamily}, Glyph=&#xea39;}"
```

Comment:
ea39 is called "ErrorBadge", that doesn't sound appropriate. "Exit" seems better.
https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font

Reply:
ChromeClose was the first icon I tried. ea39 is an x inside a circle which fit with the rest of the icons better and that is why I chose it. There wasn't a great exit button that I saw, as ChromeClose was too big and thin and the bold variant was too thick.

Reply:
Maybe this new NavItem is not neccessary. (I think) Close/Exit should be in a window (or File) menu, not a Navigation.

Reply:
I was looking for a better place, however this is the only place that is preserved across every settings pane. If there was a "file" option in a banner I would put it there.

Do you have an alternative idea as to where to put it?

Reply:
Hm, I agree it is complicated... Maybe...

- A button on the General page? Although not the prettiest solution. Possibly only visible depending on the setting ('only when needed').
- A different idea: possibly show the icon always when the Window is visible. Then it will give acces to the existing "Exit" in the menu?

Reply:
I think always showing the icon when settings is visible would be an overly complicated solution to implement (would lead to repetitive saving of settings when nothing is changing due to IPC message settings, etc) and could also lead users to believe the hide icon functionality is broken (if they check to make sure it worked or for some other reason). If they have hidden the icon, it would be counterintuitive for them to expect it to be there.

I debated putting a button on the general page, however it would only be visible there, not across all panes. That is why I placed it in the nav menu.

Showing or hiding it depending on the icon visibility setting is possible, but in my opinion, the functionality is still useful to people regardless of the setting, so keeping it might just make more sense.

Waiting to see what niels or crutkas think.

Reply:
Yeah the combination of no tray icon (menu) _and_ no window menu makes this pretty darn illogical. We need a Discussion (with the capital D) to collect user input.